src/org/ods/orchestration/InitStage.groovy

```diff
@@ -430,26 +434,30 @@ class InitStage extends Stage {
         }
     }
 
-    private void checkOutReleaseManagerRepository(def buildParams, def git,
+    private Exception checkOutReleaseManagerRepository(def buildParams, def git,
```

Comment:
I would have written the try/catch block in the method that calls this one.
This one's complexity is far away than what a normal reader can understand and introducing the try/catch blocks complicates it even more.
But this is just an opinion. You can leave it there if you consider it is better in this way.

Reply:
done